epochStart/metachain/economics.go

```diff
@@ -140,6 +146,26 @@ func (e *economics) ComputeEndOfEpochEconomics(
 	return &computedEconomics, nil
 }
 
+// compute the rewards for community - percentage from total rewards
+func (e *economics) computeRewardsForCommunity(totalRewards *big.Int) *big.Int {
+	precentageAdjustment := float64(100)
+	communityPercentage := big.NewInt(0).SetUint64(uint64(e.rewardsHandler.CommunityPercentage() * precentageAdjustment))
+	rewardsForCommunity := big.NewInt(0).Mul(totalRewards, communityPercentage)
```

Comment:
you can use core.GetPercentageOfValue instead of this.

Reply:
done